README.md

```diff
@@ -157,6 +157,11 @@ Releases are automatically built and pushed to Ansible Galaxy for any new tag. B
     1. Run `antsibull-changelog release`.
   1. Commit the changes and create a PR with the changes. Wait for tests to pass, then merge it once they have.
   1. Tag the version in Git and push to GitHub.
+  1. Run the following commands to build and release the new version on Galaxy:
+
+     ```
+     make release
```

Comment:
This isn't actually the process we'll use, at least for `community.kubernetes`. We are integrated with Zuul now, so tagging and pushing to GitHub is all that's required.

Maybe we need to add a separate section for 'Publishing new versions of `kubernetes.core` to Automation Hub'?

Reply:
certainly, I'll nuke that ... I think we should have a separate doc somewhere for the downstream release because that process isn't really a concern for the community.

Reply:
@maxamillion - As long as it's documented somewhere, I'm happy. I just know that if we don't have a documented repeatable process someone will release something broken at some point :)

(So far I haven't released anything to Automation Hub.)